toolkit/scripts/imggen.mk

```diff
@@ -105,6 +105,7 @@ $(image_package_cache_summary): $(go-imagepkgfetcher) $(chroot_worker) $(imggen_
 		--log-file=$(LOGS_DIR)/imggen/imagepkgfetcher.log \
 		--rpm-dir=$(RPMS_DIR) \
 		--tmp-dir=$(image_fetcher_tmp_dir) \
+		--toolchain-rpm-dir="$(TOOLCHAIN_RPMS_DIR)" \
```

Comment:
Let's be consistent about the plural form and use `toolchain-rpms-dir`.

Reply:
Done